examples/https-nginx/nginx-app.yaml

```diff
@@ -31,12 +31,17 @@ spec:
       - name: secret-volume
         secret:
           secretName: nginxsecret
+      - name: configmap-volume
+        configMap:
+          name: nginxconfigmap
       containers:
       - name: nginxhttps
-        image: bprashanth/nginxhttps:1.0
+        image: ymqytw/nginxhttps:1.0
```

Comment:
can you please include a liveness probe? now that you're running something else as pid1, if nginx dies nothing will notice. we want a restart if that happens.